### ISSUE TITLE
Add functionality to export / import excluded and included website match patterns

### ISSUE THREAD

**Issue by iambesi** [Midnight-Lizard/Midnight-Lizard#247]
Hello Midnight Lizard Developer (@pavel-agarkov),

I recently stumbled across this Firefox extension and started playing around with it.  I am really enjoying using it, and with the performance tweaks on your wiki, it definitely helps things to work without killing Firefox functionality.  In fact, there is only one thing that has been slightly disappointing to me, and that is, the fact that it is not possible to export / import the excluded and included websites.

That being said, there is another extension called Containerise that allows you to export / import via CSV.  It would be nice if you could do the same with Midnight Lizard.  For example, if it is a site to be excluded, you could simply have the include pattern for said site and the word ‘exclude’; or alternatively, the include pattern for the site and the schema -- e.g., Dimmed Dust, Apple Mint, Morning Mist, Increased Contrast, or similar -- for that site.

_CSV Format_

<site_pattern>, exclude|<schema_name>

Sample:

https://\*.google.com/\* , exclude
https://\*.amazon.com/\* , Increased Contrast
https://\*.diarioti.com/\* , Dimmed Dust

In short, I hope that this is a feature that we can have in a future release of the extension.  Thanks for your continuous hard work and effort on this terrific extension.

Regards,
Bernard

**Comment by pavel-agarkov** [comment]
Include and exclude rules are part of a color scheme data so it can be exported as well. Here is the instruction: https://groups.google.com/forum/#!topic/midnight-lizard-support/If-uji8fZ0w
Although you cannot import exported color schemes in Firefox since it does not support it (only Chromium based browsers do).

**Comment by pavel-agarkov** [comment]
Also in case you want to export all existing color scheme assignments to analyze them you can do the following in Firefox:
1) Navigate to about:debugging and 
![image](https://user-images.githubusercontent.com/24639631/71975174-52590a80-3224-11ea-9b60-65d563b4b249.png)
2) Find Midnight Lizard and press Debug
3) Switch to Console tab
4) Paste the following code and press Enter:
```js
browser.storage.sync.get(null).then(x=>console.log(JSON.stringify(Array.from(Object.keys(x)).filter(k=>k.startsWith('ws')).map(k=>({website:k.substr(2), scheme:x[k].colorSchemeId})),null,'  ')));
```
